classes/pluginwrapper.php

```diff
@@ -39,6 +39,8 @@
     require_once($CFG->dirroot . '/filter/wiris/classes/moodledbcache.php');
 }
 
+require_once($CFG->dirroot . '/lib/editorlib.php');
```

Comment:
Can we import it directly? Or maybe we need to do like previous imports?

```php
// This classes are shared between Wiris Quizzes and MathType
// Avoid loading twice.
if (!class_exists('moodlefilecache')) {
```

Reply:
Sé que esto es un método de protección, pero considerando que estamos trabajando en moodle y que estamos importando una librería de moodle, este archivo no debería faltar :thinking: 

Lo digo porque es fácil añadir una comprobación antes, pero también hay muchísimos archivos que se importan sin comprobar, deberíamos añadir `class_exists` a todos? @wiris/squad-mathtype-integrations

Reply:
En este caso no seria necesario